model-implementation/src/main/java/org/verapdf/model/factory/operator/OperatorFactory.java

```diff
@@ -25,17 +25,24 @@ private OperatorFactory() {
         // Disable default constructor
     }
 
-
+	/**
+	 * Transform operators and arguments of stream of pdf document,
+	 * implemented by pdfbox library, to corresponding
+	 * {@link Operator} objects of veraPDF-library.
+	 *
```

Comment:
Converts pdfbox operators and arguments from content stream to the corresponding {@link Operator} objects of veraPDF-library.